services/api-server/openapi.json

```diff
@@ -7091,15 +7091,17 @@
             "title": "Modified"
           },
           "availableCredits": {
-            "type": "number",
+            "type": "string",
             "title": "Availablecredits"
           }
         },
         "type": "object",
         "required": [
           "walletId",
           "name",
+          "description",
```

Comment:
@matusdrobuliak66 can you confirm this?

```python
class WalletGet(OutputSchema):
    wallet_id: WalletID
    name: IDStr
    description: str | None
    owner: GroupID
    thumbnail: str | None
    status: WalletStatus
    created: datetime
    modified: datetime

    model_config = ConfigDict(from_attributes=True, frozen=False)


class WalletGetWithAvailableCredits(WalletGet):
    available_credits: Decimal
```

Reply:
It seems that `description` and `thumbnail` fields appear as required in the OpenAPI Spec

Reply:
Yes I confirm, they are not required :+1: